.github/workflows/release.yml

```diff
@@ -1,14 +1,20 @@
 # **what?**
-# Take the given commit, run unit tests specifically on that sha, build and
-# package it, and then release to GitHub with that specific build (PyPi to follow later)
-
+# Release workflow provides the following steps:
+# - checkout the given commit;
+# - validate version in sources and changelog file for given version;
+# - run unit tests against given commit;
+# - build and package that SHA;
+# - release it to GitHub and PyPi with that specific build;
```

Comment:
Nit: "PyPI" rather than "PyPi" -- I realize we've been inconsistent about this in the past!
```suggestion
# - release it to GitHub and PyPI with that specific build;
```

Reply:
Fixed in https://github.com/dbt-labs/dbt-spark/pull/592/commits/517bdbf6f67d99e30b71e8c2f7fe44a849f0eaf2